src/TetherComponent.jsx

```diff
@@ -30,7 +30,7 @@ const attachmentPositions = [
 class TetherComponent extends Component {
   static propTypes = {
     renderElementTag: PropTypes.string,
-    renderElementTo: PropTypes.string,
+    renderElementTo: PropTypes.any,
```

Comment:
Should the `oneOfType` be `[PropTypes.string, PropTypes.any]`?  This is equivalent to `PropTypes.any` =P